iree/compiler/Dialect/Flow/Transforms/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ iree_cc_library(
     MLIRAnalysis
     MLIRIR
     MLIRPass
+    MLIRShapeTransforms
```

Comment:
https://source.cloud.google.com/results/invocations/804a6e03-834c-4332-b0f7-c0de879242d3/log
`/usr/bin/ld: cannot find -lMLIRShapeTransforms`
(not yet broken on `google` because bazel_to_cmake hasn't been run there yet)

I'm going to try fixing on this integrate branch instead of on `google` directly.

Reply:
Resolved, and also updated TF submodule (unrelated to that error).